cli/test/action/start.go

```diff
@@ -18,12 +18,27 @@ import (
 
 type StartCli struct {
 	stateful
+	inlineArgs    []string
+	expectedError error
 }
 
 var _ Action = (*StartCli)(nil)
 
+// Start with additional CLI arguments, and/or expecting an error (for example, if you
+// are testing with flags that should cause the command to fail)
+func StartWithArgs(args []string, expectedErr error) *StartCli {
```

Comment:
suggestion: Having two functions would remove the need to pass in `nil` in a few places, simplifying the code and making it look more deliberate.  I suggest:
```
func StartWithArgs(args []string) *StartCli {...}
func StartWithArgsE(args []string, expectedErr error) *StartCli {...}
```